recipes/sagemaker-gen-ai-jupyterlab-extension/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "sagemaker-gen-ai-jupyterlab-extension" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.8" %}
```

Comment:
```suggestion
```
Do not set this value if it is below `3.10` - `3.9` is the default, and all versions older than that are not supported anymore.